solana/tests/example-native-token-transfer.ts

```diff
@@ -101,7 +110,8 @@ describe('example-native-token-transfers', () => {
         amount,
         recipientChain: 'ethereum',
         recipientAddress: Array.from(user.publicKey.toBuffer()), // TODO: dummy
-        shouldQueue: false
+        shouldQueue: false,
+        config: await ntt.getConfig()
```

Comment:
is this necessary? the function internally will make this exact function call anyway